assets/prefabs/Equipment/Steel/SteelKatana.prefab

```diff
@@ -0,0 +1,22 @@
+{
+    "parent" : "engine:iconItem",
+    "Item" : {
+        "icon" : "Equipment:SteeKatana",
```

Comment:
Change this to `Equipment:SteelKatana` (missed an l) and rename the icon to SteelKatana.png